Cargo.toml

```
@@ -30,7 +30,7 @@ http = "0.2"
 http-body = { git = "https://github.com/hyperium/http-body" }
 httpdate = "0.3"
 httparse = "1.0"
-h2 = { git = "https://github.com/hyperium/h2", optional = true }
+h2 = { git = "https://github.com/hyperium/h2", optional = true, branch = "eliza/writev" }
```

now that hyperium/h2#500 has merged, we can switch this back to
```suggestion
h2 = { git = "https://github.com/hyperium/h2", optional = true }
```